EXPLAINER.md

```diff
@@ -0,0 +1,227 @@
+        "html/introduction.html",
+        "html/epigraph.html",
+        "html/c001.html",
+        "html/c002.html",
+        "html/c003.html",
+        "html/c004.html",
+        "html/c005.html",
+        "html/c006.html"
+    ],
+
+    "resources": [
+        "css/mobydick.css",
+        {
+            "type": "PublicationLink",
+            "rel": "https://www.w3.org/ns/wp#cover-page",
+            "url": "images/cover.jpg",
+            "encodingFormat": "image/jpeg"
+        },{
+            "type": "PublicationLink",
+            "url": "html/toc.html",
+            "rel": "contents"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneral.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBol.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBolIta.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralItalic.otf",
+            "encodingFormat": "font/otf"
+        }
+    ]
+}
+
+```
+
+Note that `readingOrder` defines the sequence of primary resources that form the publication. `resources` enumerates all the other resources that are required to render the publication. 
+
+## Design choices
+
+The unanswered question:
+
+> is the goal of this group to make a new packaging format for specialist book reading software and devices, or is it to obtain first class support for missing book-related features in the web platform as a whole?
+
+
+
+The key questions are [1] identifying the "bounds" of the publication, [2] defining the ordering of the primary resources, and [3] figuring out how to express metadata for the publication as a whole. 
+
+
+### 1. What's part of the publication?
+
+- The [Web Application Manifest spec](https://w3c.github.io/manifest/) uses `scope` to define the extent of a web application. 
+
+- Every component of an EPUB publication must be listed in an XML package file.
+
+- Web sites do not explicitly define their boundaries.
+
+- XML sometimes uses the idea of *transclusion*, where the contents of other documents can be incorporated into a parent document based on hypertext references. This concept can also be realized in HTML using iframes, HTML imports, custom elements, etc. The working group has shown no interest in this approach. 
+
+```html
+<html>
+  <head>
+    <title>Moby-Dick</title>
+  </head>
+  <body>
+    <web-publication>
+      <iframe src="c001.html">
+      <iframe src="c002.html">
+      <iframe src="c003.html">
+    </web-publication>
+  </body>
+</html>
+
+```
+
+The web publications spec has essentially adopted EPUB's approach, with an explicit list of resources. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. 
+
```

Comment:
Note that the group did consider the `scope` feature or other features like URI patterns, and I believe the feeling was that we would stick to a simple solution for this version and we may revisit this for later versions. (Again to avoid unnecessary discussions with the readers...)

Reply:
See 99ba6f6